python/lsst/ctrl/mpexec/cli/opt/options.py

```diff
@@ -76,8 +76,12 @@ def _to_int(value):
 )
 
 
-do_raise_option = MWOptionDecorator(
-    "--do-raise", help="Raise an exception on error. (else log a message and continue?)", is_flag=True
+pdb_option = MWOptionDecorator(
+    "--pdb",
+    help="Drop into pdb on exception (only if single process)?",
```

Comment:
Do we need a question mark here? 
Can you also expand it a bit and say that it can specify different debugger, e.g. ipdb?